application/models/User.php

```diff
@@ -540,13 +540,8 @@ public function getManagementButtons()
             'linkAttributes'   => [
                 'data-href' => $editUrl,
             ],
-            'enabledCondition' =>
-                ($permission_superadmin_read
-                    && $this->uid != 1
-                )
-                || (!$permission_superadmin_read
-                    && $this->canEdit(App()->session['loginID'])
-                )
+            'enabledCondition' => $this->canEdit()
+                                && $this->uid != App()->user->getId()
```

Comment:
This rule should be inside the userManager->canEdit() ?

Reply:
Actually, I am seeing below, isn't there already? Then, is this necesarry?

Can we just call userManager->canEdit() ?

Reply:
No , see https://github.com/LimeSurvey/LimeSurvey/pull/3495#discussion_r1338619193

Currently button his hidden for self.

Reply:
Adding comment is a good idea : let's go.